samples/attested_tls/non_enc_client/client.cpp

```diff
@@ -247,18 +247,17 @@ int main(int argc, char** argv)
         goto done;
     }
 
-    if ((ctx = SSL_CTX_new(SSLv23_client_method())) == nullptr)
+    if ((ctx = SSL_CTX_new(TLS_client_method())) == nullptr)
     {
         printf(TLS_CLIENT "TLS client: unable to create a new SSL context\n");
         goto done;
     }
 
-    // choose TLSv1.2 by excluding SSLv2, SSLv3 ,TLS 1.0 and TLS 1.1
+    // Exclude SSLv2, SSLv3, TLS 1.0, TLS 1.1 and TLS 1.2
     SSL_CTX_set_options(ctx, SSL_OP_NO_SSLv2);
     SSL_CTX_set_options(ctx, SSL_OP_NO_SSLv3);
     SSL_CTX_set_options(ctx, SSL_OP_NO_TLSv1);
     SSL_CTX_set_options(ctx, SSL_OP_NO_TLSv1_1);
```

Comment:
```suggestion
    SSL_CTX_set_options(ctx, SSL_OP_NO_TLSv1_1);
    SSL_CTX_set_options(ctx, SSL_OP_NO_TLSv1_2);
```
Make code match the comment in line 256

Reply:
`SSL_OP_NO_TLSv1_2` is intentionally excluded. As there is one use case where Openssl client communicates with mbedtls server. Looks like this communication is happening on TLS V1.2. If TLS V1.2 is excluded here it is resulting in communication failure.

Reply:
In this case, I'd suggest using the previous comment.

Reply:
Can't the non-enclave client also uses the recommended configuration?